src/Components/QuizEncouragement.tsx

```diff
@@ -61,23 +144,24 @@ const QuizButton = styled.div`
   align-self: flex-start;
   background-color: ${(props) => props.theme.colours.blue};
   display: inline-block;
+
+  ${(props) => props.theme.screenSizes.tabletPortraitPlus} {
+    padding: 0.5rem 3.2rem;
+  }
 `
 
 export const QuizEncouragement = (): JSX.Element => {
   return (
     <Link href={`/quiz`} passHref>
       <QuizCard>
         <Content>
-          <h2>Want more personalised results?</h2>
+          <h2>Summer Activities</h2>
           <VerticalSpacing size={1} />
-          <QuizButton>Take our quiz</QuizButton>
+          <QuizButton>See what&apos;s on</QuizButton>
           <ImageContainer>
-            <Image
-              src="/img/try-quiz.png"
-              alt={''}
-              layout="fill"
-              objectFit="contain"
-            />
+            <img src="/img/summer_activities_mask.png" alt="" />
```

Comment:
You could use the NextJS `Image` component, which is super cool and does things like lazy loading images when users internet is being slow - https://nextjs.org/docs/api-reference/next/image